docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -297,40 +289,13 @@ class ComponentExample extends React.Component<ComponentExampleProps, ComponentE
     )
   }
 
-  private renderSourceCode = _.debounce(() => {
```

Comment:
no debouncing anymore?

Reply:
Yes, we don't need now it fact. Performance is good enough.